packages/evergreen-component-mapper/src/text-field/text-field.tsx

```diff
@@ -1,25 +1,33 @@
 import React from 'react';
 
-import { TextInputField } from 'evergreen-ui';
+import { Autocomplete, TextInputField } from 'evergreen-ui';
 
 import useFieldApi from '@data-driven-forms/react-form-renderer/use-field-api';
 import { UseFieldApiProps } from '@data-driven-forms/react-form-renderer';
 
 export interface TextFieldProps extends UseFieldApiProps<string> {
-    name: string;
-    isRequired?: boolean;
-};
+	name: string;
+	items: string[];
+	isRequired?: boolean;
+}
 
 const TextField: React.FC<TextFieldProps> = (props) => {
-	const { input, meta, isRequired, ...rest } = useFieldApi(props);
+	const { input, meta, isRequired, items, ...rest } = useFieldApi(props);
```

Comment:
Are `rest` props the same for `TextInputField` and `Autocomplete`?

Reply:
They are not the same. I will probably have to put the `rest` props on the `TextInputField` which should still work. I will apply it to the wrapped `TextInputField` instead.

For the `Autocomplete`, the `items` prop (and the `children` prop, which is already handled by wrapping the text input component) is only required but we can provide an empty array if it wasn't provided (based on the change I will do for the above comment).

Reply:
Also, for your information, some of the props do get passed down from `Autocomplete` to the wrapped `TextInputField` via the `getInputProps` function you see in the code. Therefore, the `input` and some other important DDF props should be passed down properly.